salt/metalk8s/salt/master/certs/init.sls

```diff
@@ -8,8 +8,8 @@
 # * salt-api              -> generate Salt API key and certificate
 #
 include:
-  # Some server certs are required to be published in K8s API by Salt master
```

Comment:
Why remove the comment? Since these aren't actual certs required by Salt master itself, it explains why they are here.